rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/TreeView.java

```diff
@@ -194,54 +193,27 @@ public void onNodeContextClick(NodeContextClickEvent event) {
                 menu.setShadowDepth(10);
 
                 MenuItem expandItem = new MenuItem("Expand all", Images.instance.expand_16().getSafeUri().asString());
-                expandItem.addClickHandler(new ClickHandler() {
-                    @Override
-                    public void onClick(MenuItemClickEvent event) {
-                        expandAll();
-                    }
-                });
+                expandItem.addClickHandler(event17 -> expandAll());
 
                 MenuItem collapseItem = new MenuItem("Collapse all",
                                                      Images.instance.close_16().getSafeUri().asString());
-                collapseItem.addClickHandler(new ClickHandler() {
-                    @Override
-                    public void onClick(MenuItemClickEvent event) {
-                        closeAll();
-                    }
-                });
+                collapseItem.addClickHandler(event16 -> closeAll());
 
                 MenuItem removeItem = new MenuItem("Remove",
                                                    RMImages.instance.node_remove_16().getSafeUri().asString());
-                removeItem.addClickHandler(new ClickHandler() {
-                    @Override
-                    public void onClick(MenuItemClickEvent event) {
-                        controller.removeNodes();
-                    }
-                });
+                removeItem.addClickHandler(event15 -> controller.removeNodes());
 
                 MenuItem lockItem = new MenuItem("Lock", lockItemImageResource);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)